SYCL/ESIMD/lsc/Inputs/lsc_surf_load.hpp

```diff
@@ -17,11 +17,11 @@ using namespace sycl;
 using namespace sycl::ext::intel::esimd;
 using namespace sycl::ext::intel::experimental::esimd;
 
-template <int case_num, typename T, uint32_t Groups, uint32_t Threads,
-          uint16_t VL, uint16_t VS, bool transpose,
-          lsc_data_size DS = lsc_data_size::default_size,
-          cache_hint L1H = cache_hint::none, cache_hint L3H = cache_hint::none,
-          bool use_prefetch = false>
+template <
+    int case_num, typename T, uint32_t Groups, uint32_t Threads, uint16_t VL,
+    uint16_t VS, bool transpose, lsc_data_size DS = lsc_data_size::default_size,
+    cache_hint L1H = cache_hint::none, cache_hint L3H = cache_hint::none,
+    bool use_prefetch = false, typename Flags = __ESIMD_NS::overaligned_tag<4>>
```

Comment:
The lines 64 and 65 below need to use std::vector constructor accepting alignment to guarantee it is at least 8 even for T=int8_t